packages/paste-website/src/component-examples/StatusPatternExamples.tsx

```diff
@@ -0,0 +1,511 @@
+            <ProcessSuccessIcon
+              color="colorTextIconSuccess"
+              decorative={false}
+              title="active"
+            />
+            <Text as="div">Active</Text>
+          </Box>
+        </Td>
+        <Td>2020-10-15</Td>
+      </Tr>
+      <Tr>
+        <Td>90294808908872345</Td>
+        <Td>WE928471c21gjioGasd</Td>
+        <Td>North America</Td>
+        <Td>
+          <Box display="inline-flex" columnGap="space20">
+            <ProcessNeutralIcon
+              color="colorTextIconNeutral"
+              decorative={false}
+              title="active"
+            />
+            <Text as="div">Ready</Text>
+          </Box>
+        </Td>
+        <Td>2020-10-15</Td>
+      </Tr>
+    </TBody>
+  </Table>
+</>
+`.trim();
+
+export const tableWithPlainTextExample = `
+<>
+  <Heading as="h2" variant="heading20">
+    Reboots
+  </Heading>
+
+  <Table>
+    <THead>
+      <Tr>
+        <Th>Title</Th>
+        <Th>Media</Th>
+        <Th>Status</Th>
+      </Tr>
+    </THead>
+    <TBody>
+      <Tr>
+        <Td>Bewitched</Td>
+        <Td>Series</Td>
+        <Td>
+          <Box display="flex">
+            <ProcessWarningIcon color="colorTextIconWarning" decorative={false} title="progress halted" />
+            <Text marginLeft="space20">Progress Halted</Text>
+          </Box>
+        </Td>
+      </Tr>
+      <Tr>
+        <Td>Mulan</Td>
+        <Td>Feature</Td>
+        <Td>Released</Td>
+      </Tr>
+      <Tr>
+        <Td>Little Women</Td>
+        <Td>Feature</Td>
+        <Td>Released</Td>
+      </Tr>
+    </TBody>
+  </Table>
+</>
+`.trim();
+
+export const linkInStatusExamples = `
+<Box display="flex" columnGap="space80" rowGap="space60" flexWrap="wrap">
+  <Box display="flex">
+    <ProcessErrorIcon
+      color="colorTextIconError"
+      decorative={false}
+      title="error"
+    />
+    <Anchor
+      marginLeft="space20"
+      href="#"
+      showExternal
+    >
+      Pre-registration needed
+    </Anchor>
+  </Box>
+  <Box display="inline-flex" columnGap="space20">
+    <ProcessErrorIcon
+      color="colorTextIconError"
+      decorative={false}
+      title="error"
+    />
+    <Box>
+      <Text as="div">Pre-registration needed</Text>
+      <Box>
+        <Anchor
+        marginLeft="space20"
+        href="#"
+        showExternal
+        >
+          Fill out the registration form here
+        </Anchor>
+      </Box>
+    </Box>
+  </Box>
+  <Box display="inline-flex" columnGap="space20">
+    <ProcessErrorIcon
+      color="colorTextIconError"
+      decorative={false}
+      title="error"
+    />
+    <Box>
+      <Text as="div">Pre-registration needed</Text>
+      <Box>
+        <Button
+        marginLeft="space20"
+        variant="link"
+        >
+          View details
+        </Button>
+      </Box>
+    </Box>
+  </Box>
+</Box>
+`.trim();
+
+export const otherCommonPlacements = `
+<Stack orientation="vertical" spacing="space60">
+  <MediaObject>
```

Comment:
Should probably cut this example for now, since we're adding Connectivity statuses back in later.

Reply:
💯